addons/jump-to-def/userscript.js

```diff
@@ -27,24 +43,26 @@ export default async function ({ addon, msg, console }) {
       let block = Blockly.registry ? this.startBlock : this.startBlock_;
       for (; block; block = block.getSurroundParent()) {
         if (block.type === "procedures_call") {
-          let findProcCode = block.getProcCode();
-
-          let topBlocks = addon.tab.traps.getWorkspace().getTopBlocks();
-          for (const root of topBlocks) {
-            if (root.type === "procedures_definition") {
-              let label = root.getChildren()[0];
-              let procCode = label.getProcCode();
-              if (procCode && procCode === findProcCode) {
-                // Found... navigate to it!
-                utils.scrollBlockIntoView(root);
-                return;
-              }
-            }
-          }
+          jumpToBlockDefinition(block);
+          return;
         }
       }
     }
 
     _doBlockClick_.call(this);
   };
+
+  addon.tab.createBlockContextMenu(
+    (items, block) => {
+      if (!addon.self.disabled && block.type === "procedures_call") {
+        items.splice(3, 0, {
```

Comment:
This puts the new item before Edit in the code area and after Edit in the flyout:

![image](https://github.com/user-attachments/assets/b4b6e711-b9b2-4514-8756-677d7ddd68c5)
![image](https://github.com/user-attachments/assets/6ec87c52-12af-4e10-a807-6fe4048e4581)

I think it should always be after Edit, like in Scratch 2.0:

![image](https://github.com/user-attachments/assets/35b83608-8fbd-456f-97df-594b8b020a8e)